mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/FieldProjection.java

```diff
@@ -41,9 +41,9 @@
 
 	/**
 	 * @return A value controlling how the data fetched from the backend should be converted.
-	 * @see ValueConvert
-	 * @see SearchProjectionFactory#field(String, Class, ValueConvert)
+	 * @see ProjectionValueConvert
+	 * @see SearchProjectionFactory#field(String, Class, ProjectionValueConvert)
 	 */
-	ValueConvert convert() default ValueConvert.YES;
+	ProjectionValueConvert convert() default ProjectionValueConvert.MAPPING;
```

Comment:
I'm not sure that there's a nice way to "deprecated" things for this annotation 😕